sale_automatic_workflow/data/automatic_workflow_data.xml

```diff
@@ -44,6 +44,7 @@
         <field name="name">Automatic</field>
         <field name="picking_policy">one</field>
         <field name="validate_order" eval="1" />
+        <field name="send_order_confirmation_mail" eval="1" />
```

Comment:
As this is a new optional feature being introduced, wouldn't it be better to disable it by default?
```suggestion
        <field name="send_order_confirmation_mail" eval="0" />
```

Reply:
IMO it is not necessary to leave it at 0 by default because this is noupdate=1